runner/src/indexer/indexer.ts

```diff
@@ -14,13 +17,15 @@ interface Dependencies {
   provisioner: Provisioner
   DmlHandler: typeof DmlHandler
   parser: Parser
+  IndexerLogger: typeof IndexerLogger
 };
 
 interface Context {
   graphql: (operation: string, variables?: Record<string, any>) => Promise<any>
   set: (key: string, value: any) => Promise<any>
   debug: (...log: any[]) => Promise<void>
   log: (...log: any[]) => Promise<void>
+  warn: (...log: any[]) => Promise<void>
   error: (...log: any[]) => Promise<void>
```

Comment:
```suggestion
  debug: (message: string) => Promise<void>
  log: (message: string) => Promise<void>
  warn: (message: string) => Promise<void>
  error: (message: string) => Promise<void>
```

The variadic arguments make the types really awkward - we need to use `any` as we can't guarantee what is passed in. I'm guessing these are only variadic because the original log function was. Since we are adding new methods it may be a good opportunity to just have a single typed argument like so.